test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -65,10 +47,6 @@ public class MutableObjectModelBinderTest
                         MetadataProvider = metadataProvider,
                         ValidatorProvider = Mock.Of<IModelValidatorProvider>(),
                     },
-
-                    // CanCreateModel() ignores the BinderModelName and ModelName properties.
```

Comment:
This code was intended to ensure these properties do /not/ affect `CanCreateModel()`. Please restore the corresponding parameters and inline data.